apps/frontend/src/components/xpress/XpressProposalTable.tsx

```diff
@@ -0,0 +1,363 @@
+import MaterialTable, { Column, OrderByCollection } from '@material-table/core';
+import { t, TFunction } from 'i18next';
+import React, { useEffect, useState } from 'react';
+import { useSearchParams } from 'react-router-dom';
+
+import { ProposalsFilter } from 'generated/sdk';
+import { useCallsData } from 'hooks/call/useCallsData';
+import { useProposalStatusesData } from 'hooks/settings/useProposalStatusesData';
+import { useTechniquesData } from 'hooks/technique/useTechniquesData';
+import { StyledContainer, StyledPaper } from 'styles/StyledComponents';
+import {
+  addColumns,
+  fromArrayToCommaSeparated,
+  setSortDirectionOnSortField,
+} from 'utils/helperFunctions';
+import { tableIcons } from 'utils/materialIcons';
+
+import { ProposalViewData, useProposalsCoreData } from './useProposalsCoreData';
+import { useXpressInstrumentsData } from './useXpressInstrumentsData';
+import XpressProposalFilterBar from './XpressProposalFilterBar';
+
+const XpressProposalTable = () => {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [selectedProposals, setSelectedProposals] = useState<
+    ProposalViewData[]
+  >([]);
+  const { techniques, loadingTechniques } = useTechniquesData();
+  const { proposalStatuses, loadingProposalStatuses } =
+    useProposalStatusesData();
+  const [currentPage, setCurrentPage] = useState(0);
+  const [rowsPerPage, setRowsPerPage] = useState(20);
+  const PREFETCH_SIZE = 200;
```

Comment:
Will this not be best be server side rather than use prefetching we can get slices based on page size

Reply:
@mutambaraf, yes we can do it. Shall we handle it as part of another issue, given that this won't cause any issue now and considering the target date?

Reply:
I think this is just passing two additional filters to the backend.

Reply:
Sure, will include it.